### PR TITLE
Added graph to Blend Pool page

### PR DESCRIPTION
**Pull request by IanWoodard**
Added an interactive graph to the blend pool page to help visualize the pool's returns compared to Uniswap V2 and 50/50 HODL.
While the graph is fully functional, it does not have a source of data yet so that data is randomly generated for the time being.

Images:
graph-example5
graph-example6

### CODE REVIEW

**Comment by IanWoodard** [comment]
Updated the graph as well as the entire blend pool page to fit the spec. There is still more work to be done to fully fit the spec, but those changes are not in the scope of this PR and will be done separately.

fullgraph

**Comment by IanWoodard** [comment]
Additionally, I added a bit more responsiveness than the page had before. There is still some tweaking that needs to be done, especially for the mobile viewports.

File: src/util/Dates.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    : subMinutes(updatedDate, Math.abs(timezoneDifference));
+}
+
+/**
+ * Note: Only works if the dates are equidistant apart
+ * @param dates Array of dates
+ * @param n number of dates to return (+/- 1)
+ * @returns Array of dates (as string) that are evenly spaced throughout the given array of dates
+ */
+export function getEvenlySpacedDates(dates: string[], n: number) {
```
Comment (adamegyed) [inline src/util/Dates.ts]: Annotate return type with `: string[]`

File: src/util/Dates.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { add, addMinutes, subMinutes } from 'date-fns/esm';
+
+/**
+ *
+ * @param date a date
+ * @param step a duration to add to the date
+ * @returns the date plus the duration that ignores timezone differences
+ */
+export function agnosticAdd(date: Date, step: Duration) {
```
Comment (adamegyed) [inline src/util/Dates.ts]: Annotate return type with `: Date`

File: src/data/constants/Breakpoints.ts [hunk 3]
```diff
@@ -1,2 +1,3 @@
+export const RESPONSIVE_BREAKPOINT_LG = '1400px';
 export const RESPONSIVE_BREAKPOINT_MD = '1200px';
 export const RESPONSIVE_BREAKPOINT_SM = '768px';
```
Comment (adamegyed) [inline src/data/constants/Breakpoints.ts:3]: We should find a build process step to consolidate these constants with the Tailwind breakpoints to reduce duplication. Good to merge for this PR though.

**Review by adamegyed — APPROVED** [review]
Good to merge after a couple small changes. As with most of these PRs, we'll need to do additional work integrating into future components.

**Pull request merged**